src/rust/bitbox02-rust-c/src/util.rs

```diff
@@ -11,6 +11,12 @@ pub extern "C" fn rust_util_zero(mut dst: BytesMut) {
     util::zero(dst.as_mut())
 }
 
+#[no_mangle]
+pub extern "C" fn rust_util_all_ascii_bytes(bytes: Bytes) -> bool {
+    assert!(!bytes.buf.is_null());
```

Comment:
this isn't needed, if the pointer is null and the length != 0 it will hit an assert in the `as_ref()` implementation. If the pointer is null and the length is 0 it should be `true` to call `all_ascii_bytes()`.

Reply:
Done